dependencies.yml

```diff
@@ -176,11 +176,8 @@ com.spotify:
       to: com.linecorp.armeria.internal.shaded.futures
   futures-extra: { version: '4.3.1' }
 
-# The OkHttp version is aligned with Retrofit's transitive version
-# to avoid unexpected conflicts. This is a temporary measure that
-# will be improved with better dependency version management in the future.
```

Comment:
Could you put the sentence back which is `# OkHttp is used only for testing in it:okhttp module.`
so that we can easily write the commit message of dependency updates?

Reply:
Done 👍